common/lib/capa/capa/inputtypes.py

```diff
@@ -325,12 +326,15 @@ def _get_render_context(self):
             'describedby_html': '',
         }
 
-        # Don't add aria-describedby attribute if there are no descriptions
-        if self.response_data.get('descriptions'):
-            description_ids = ' '.join(self.response_data.get('descriptions').keys())
-            context.update(
-                {'describedby_html': 'aria-describedby="{}"'.format(description_ids)}
-            )
+        # Generate the list of ids to be used with the aria-describedby field.
+        # Every list should contain the status id
+        status_id = 'status_' + self.input_id
+        descriptions = list([status_id])
+        descriptions.extend(self.response_data.get('descriptions', {}).keys())
+        description_ids = ' '.join(descriptions)
+        context.update(
+            {'describedby_html': 'aria-describedby="{}"'.format(description_ids)}
```

Comment:
Thank you for taking a look.  If it is easier you can focus on my most recent commit 3fd76be2598e640a707ca5b5f6be22271793699f it has all of the safecommit changes I am concerned about.

Christina and Alisan have been looking at the python and front end changes in detail.

I will make and test this change.

Also, I am slightly confused by when to use HTML And Text, but I will ask you in HipChat.